docs/en/observability/monitor-uptime-synthetics.asciidoc

```diff
@@ -1,50 +1,92 @@
 [[monitor-uptime-synthetics]]
-= Synthetic monitoring via the {uptime-app}
+= Uptime and synthetic monitoring
 
 ++++
-<titleabbrev>Synthetic monitoring</titleabbrev>
+<titleabbrev>Uptime and synthetic monitoring</titleabbrev>
 ++++
 
-Synthetic monitoring extends traditional end-to-end testing techniques because it allows your tests to run continuously on the cloud.
-With Elastic's Synthetic Monitoring product, you can assert that your application continues to work after a deployment by reusing the same journeys that you used to validate the software on your machine.
+[[uptime-monitors]]
 
-You can use synthetic monitors to detect bugs caused by invalid states you couldn't anticipate and didn't write tests for.
-Synthetic monitors can also help you catch bugs in features that don't get enough traffic by allowing you to periodically simulate users' actions.
+The {uptime-app} uses {agent} to check the status of your services and applications periodically.
+By adding the Elastic Synthetics integration to the policy used by your agent, you can monitor the
+availability of network endpoints and services using the following Uptime monitors:
 
-The {uptime-app} in {kib} includes two types of synthetic monitoring:
-
-* <<monitoring-synthetics>>
 * <<monitoring-uptime>>
+* <<monitoring-synthetics>>
+
+[discrete]
+[[monitoring-uptime]]
+== Lightweight HTTP/S, TCP, and ICMP monitors
+
+In the {uptime-app}, you can monitor the status of network endpoints using the following lightweight checks:
+
+|===
+
+| *ICMP monitor* | Check the availability of your hosts. The ICMP monitor uses ICMP (v4 and v6) Echo
```

Comment:
Can we flip this order? HTTP is the most common check, followed by perhaps a tie for TCP and ICMP, however I think it makes sense for TCP to be after HTTP because that puts the protocols in descending order of complexity.